### DISCUSSION TITLE
Superconductivity in MOOSE

### DISCUSSION THREAD

**Discussion by gj19866** [idaholab/moose#27413] [Show and tell]
Hi there,
I just wanted to say thank you for all the assistance from the MOOSE team, especially @GiudGiud, in the fruition of my current research.
It has been great to see MOOSE applied to the field of superconductivity.

All the best,
Gillian

[Final_report_Final_checks (2).pdf](https://github.com/idaholab/moose/files/15031370/Final_report_Final_checks.2.pdf)

**Comment by GiudGiud** [comment]
Thanks for sharing! We are glad to see people pushing MOOSE into new fields!

If you made an app or a repo feel free to share a link for posterity. I may have missed it but I didn't see it in your report.
